collect_app/src/androidTest/java/org/odk/collect/android/espressoutils/pages/ErrorDialog.java

```diff
@@ -5,19 +5,21 @@
 import org.odk.collect.android.R;
 
 public class ErrorDialog extends Page<ErrorDialog> {
+    private final OkDialog okDialog;
```

Comment:
I really like the the `OkDialog`. Seems like the best way to verify "any" dialog at the moment. 

For me, it does feel like the `ErrorDialog` should inherit from the `OkDialog` rather than contain one. What do you think?

Reply:
I feel like I had a reason but I don't even know anymore.